v-next/hardhat-ignition-core/src/types/artifact.ts

```diff
@@ -54,7 +70,7 @@ export interface BuildInfo {
   solcVersion: string;
   solcLongVersion: string;
   input: CompilerInput;
-  output: CompilerOutput;
+  userSourceNameMap: Record<string, string>;
```

Comment:
Do we no longer have the output in the build info?

Reply:
Correct, this is the updated version of BuildInfo from Hardhat v3.